recipes/poliastro/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install --no-deps .
```

Comment:
In fact this works just fine, I've been using this recipe for a while already:

https://github.com/poliastro/poliastro/commit/9d25954fed9652e232d1159d0f3a5d00039a04c1

I don't know what is the current (March 2016) consensus on this, but half a year ago this thread on numpy-dev prompted me to change that line:

https://mail.scipy.org/pipermail/numpy-discussion/2015-October/074086.html

Of course I can restore it to its previous state.